lib/undercover/plugin.rb

```diff
@@ -24,13 +24,22 @@ class DangerUndercover < Plugin
     # If there are reports then it shows the report as a warning in danger.
     # @return  [void]
     #
-    def report(undercover_path = DEFAULT_PATH, sticky: true)
+    def report(undercover_path = DEFAULT_PATH, sticky: true, in_line: false, fail_0_coverage: false)
```

Comment:
Is `fail_0_coverage` deliberately vague? It's not clear from the name whether it fails when any line has 0 coverage, any method has 0 coverage or the entire PR has 0 coverage (actually I initially thought it's the last one, but from the code it doesn't appear that way)

Reply:
Open to suggestions on the name of this variable! :D I don't find it `vague` but I do find it creepy myself. Can you think of a better naming with a fresh point of view?

Reply:
After an offline chat where you clarified that `fail_0_coverage` means that it fails unless every line is tested, my suggestion is something like: 
- `fail_on_missing_coverage`
- `fail_if_anything_untested`
- `fail_on_untested_line`

Reply:
`fail_on_missing_coverage` sounds good to me :) will use that

Reply:
```suggestion
    def report(undercover_path = DEFAULT_PATH, sticky: true, in_line: false, fail_on_missing_coverage: false)
```